api/Services/JsonResponseService.js

```diff
@@ -0,0 +1,20 @@
+let unsuccessful = () => {
+    return {
+        "success": false,
+        "message": "",
+        "status": "",
```

Comment:
Use a default status code here. Perhaps 500 as it is a general server error code

Reply:
Just in case this function is ever used & no status code manually entered